airtable/plugin.py

```diff
@@ -2,11 +2,42 @@
 from dataclasses import dataclass
 from datetime import datetime
 from typing import Any
+import re
 
 import httpx
 
 from lutraai.augmented_request_client import AugmentedTransport
 
+@dataclass
+class AirtablePath:
+    base_id: str
+    table_id_or_name: str
+
+def extract_airtable_path_info(url: str) -> AirtablePath:
```

Comment:
```suggestion
@dataclass
class AirtableIDs:
    base_id: str
    table_id: str
    record_id: str

def extract_airtable_ids(url: str) -> AirtableIDs:
    # Make this extract all IDs.
```

I suggest these changes because:
* Airtable "path" is not really a concept anywhere.
* We're probably going to want to be able to extract record IDs at some point for other APIs.